src/components/Header.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { Navigation } from 'hds-react';
+
+import { useKeycloak } from '../clients/keycloak';
+// import { useTranslation } from 'react-i18next';
+
+type LanguageOption = { code: string; label: string };
+
+const Header = (): React.ReactElement => {
+  const languageOptions = [
+    { code: 'fi', label: 'Suomi' },
+    { code: 'sv', label: 'Svenska' },
+    { code: 'en', label: 'English' }
+  ];
+  /*
+  const { i18n } = useTranslation();
+  const setLanguage = (code: string) => {
+    i18n.changeLanguage(code);
+  };
+  */
+  const keycloak = useKeycloak();
+  const authenticated = keycloak.isAuthenticated();
+  const initialized = keycloak.isInitialized();
+  const user = keycloak.getUser();
+
+  const [language, setLanguage] = useState(languageOptions[0]);
+  const [active, setActive] = useState<'link' | 'button' | 'dropdown'>();
+
+  // show helsingfors logo if swedish is selected as the language
+  const logoLanguage = language.code === 'sv' ? 'sv' : 'fi';
+
+  const title =
+    language.code === 'sv'
+      ? 'Helsingfors Systembolaget'
+      : 'Helsinki Design System';
```

Comment:
Just for the looks, maybe use proper titles?

Reply:
Removed.